src/main/java/org/apache/maven/plugins/shade/mojo/ShadeMojo.java

```diff
@@ -606,7 +607,8 @@ private void processArtifactSelectors( Set<File> artifacts, Set<String> artifact
         {
             if ( !artifactSelector.isSelected( artifact ) )
             {
-                getLog().info( "Excluding " + artifact.getId() + " from the shaded jar." );
+                String exclusionMessage = "Excluding " + artifact.getId() + " from the shaded jar.";
+                getLog().info( MessageUtils.buffer().warning( exclusionMessage ).toString() );
```

Comment:
Maybe I misunderstand a thing but if used `MessageUtils.buffer().warning( exclusionMessage ).toString()` shouldn't than the `getLog().warn`  instead of `getLog().info`?